### ISSUE TITLE
The color doesn't change

### ISSUE THREAD

**Issue by Neha--Agarwal** [makovkastar/FloatingActionButton#203]
Even when I change the color of button in its XML code, the color of button is keeping the same, blue. Why is it so? Please help!

**Comment by Flaff** [comment]
make sure you are editing
fab:fab_colorNormal
in your xml layout.

**Comment by MohammadAbbasKhan** [comment]
facing same problem any answer

**Comment by JackLauu** [comment]
Yes, the color only could be changed by code,  can't get it from xml. How to solve it ?

**Comment by Iamtodor** [comment]
@JackLauu `fab:fab_colorNormal="@color/fab_background"` this works for me. Don't know how to change item's color inside, for example: I use android:src="@android:drawable/ic_input_add" I want to change + color..

**Comment by xtonousou** [comment]
To change color guys you should declare namespace
`xmlns:fab="http://schemas.android.com/apk/res-auto"`
and not
`xmlns:fab="http://schemas.android.com/tools"`

After you do that, you can change the color programmatically or with xml.
